packages/sdk/src/Incentives/ERC20VariableCriteriaIncentive.ts

```diff
@@ -319,13 +351,15 @@ export function gasRebateIncentiveCriteria(): IncentiveCriteria {
  * @param {Address} param0.asset - The address of the ERC20 asset to incentivize.
  * @param {bigint} param0.reward - The reward amount to distribute per action.
  * @param {bigint} param0.limit - The total limit of the asset distribution.
+ * @param {bigint} param0.maxReward - The maximum value claimable from a single completion.
  * @param {IncentiveCriteria} param0.criteria - The incentive criteria for reward distribution.
  * @returns {Hex}
  */
 export function prepareERC20VariableCriteriaIncentivePayload({
   asset,
   reward,
   limit,
+  maxReward = 0n,
```

Comment:
Should we default the `maxReward` to the value of `reward`?

Reply:
[If maxReward is zero it'll be ignored](https://github.com/boostxyz/boost-protocol/pull/245/files#diff-d49612957cd995a8ab9ca7131b0c81c7b906f6e9b59c3343e6d23164218ff628R67)